hera_mc/cm_sysdef.py

```diff
@@ -0,0 +1,128 @@
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2019 the HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+"""
+This module defines all of the system parameters for hookup.  It tries to
+contain all of the ad hoc messy part of walking through a signal chain to
+this one file.
+"""
+
+from __future__ import absolute_import, division, print_function
+from . import cm_utils
+
+
+checking_order = ["parts_hera", "parts_rfi", "parts_paper", "parts_test"]
+full_connection_path = {"parts_paper": ["station", "antenna", "feed", "front-end",
+                                        "cable-feed75", "cable-post-amp(in)",
+                                        "post-amp", "cable-post-amp(out)",
+                                        "cable-receiverator", "cable-container",
+                                        "f-engine"],
+                        "parts_hera": ["station", "antenna", "feed", "front-end",
+                                       "cable-rfof", "post-amp", "snap", "node"],
+                        "parts_rfi": ["station", "antenna", "feed", "temp-cable",
+                                      "snap", "node"],
+                        "parts_test": ["vapor"]
+                        }
+corr_index = {"parts_hera": 5, "parts_paper": 9, "parts_rfi": 3}
+all_pols = ["e", "n"]
+redirect_part_types = ['node']
+
+
+def handle_redirect_part_types(part):
+    print("Redirected parts require that you determine the set of parts externally and re-run.")
+    print("\t\tFor this part type:  'parts.py conn -p {}'".format(part.hpn))
+    print("\t\tThen re-run hookup with a csv list of parts.")
```

Comment:
I'm not sure I really understand this. If all you're going to do is rerun using the list passed out by the suggested command, why not do it automatically in the code (rather than having a human in the loop)?

Reply:
This should be an issue in the future, since it is not straightforward in this scope.  I would like to implement that function, but it will need a bit of thought/coding.  For now, the "helpful" message may "help" (to remind me at least).  It also tells the user that the request won't be complete.

Reply:
ok. Do you want to make an issue?

Reply:
I updated issue #328